src/raftstore/store/debug.rs

```diff
@@ -219,6 +220,49 @@ impl Debugger {
         compact_range(db, handle, start, end, false);
         Ok(())
     }
+
+    /// Set a region to tombstone by manual, and apply other status(such as
+    /// peers, version, and key range) from `region` which comes from PD normaly.
+    pub fn set_region_tombstone(&self, id: u64, region: Region) -> Result<()> {
+        let store_id = self.get_store_id()?;
+        if region
+            .get_peers()
+            .iter()
+            .any(|peer| peer.get_store_id() == store_id)
+        {
+            return Err(box_err!("The peer is still in new peers list"));
```

Comment:
What if there is a new replica (same store id but different region id) on this store?

Reply:
We compare this peer's store_id with peers having the specified region. So I think if `remove-peer` really finished, we can got `false` here, which is expected.